### PR TITLE
Update how to run integration tests command

### PR DESCRIPTION
**Pull request by gciavarrini**
To reflect changes in PR https://github.com/parodos-dev/parodos/pull/376

Depends on https://github.com/parodos-dev/parodos/pull/376

### CODE REVIEW

**Review by pkliczewski — COMMENTED** [review]
/lgtm

File: documentation/latest/dev/guide.md [hunk 1]
```diff
@@ -70,24 +70,35 @@ kube-system          kube-scheduler-kind-control-plane            1/1     Runnin
...
 
-Finally, retrieve and export the cluster IP address:
+Retrieve and export the cluster IP address:
 
 ```bash
 export SERVER_IP=$(kubectl get nodes kind-control-plane -o json  |  jq -r '[.status.addresses[] | select(.type=="InternalIP")] | .[0].address')
 ```
 
+Finally, use the value of `$SERVER_IP` to configure services hosts in your
+`/etc/hosts` file:
```
Comment (masayag) [inline documentation/latest/dev/guide.md:80]: @gciavarrini I think this is the missing piece that prevented me from working with kind without port forwarding.

File: documentation/latest/dev/guide.md [hunk 2]
```diff
@@ -70,24 +70,35 @@ kube-system          kube-scheduler-kind-control-plane            1/1     Runnin
...
+NOTIFICATION_SERVICE_HOST=notification-service.parodos-dev \
 NOTIFICATION_SERVER_PORT=8081 \
 SERVER_PORT=80 \
 mvn verify -pl integration-tests -P integration-test -Dspring.profiles.active=dev
 ```
 
-Please note that `WORKFLOW_SERVICE_PATH` and `NOTIFICATION_SERVICE_PATH` are
-the `Ingress` paths defined in [ingress.yaml](https://github.com/parodos-dev/parodos/blob/main/hack/manifests/testing/ingress.yaml).
+Please note that `WORKFLOW_SERVICE_HOST` and `NOTIFICATION_SERVICE_HOST` are
+the `Ingress` hosts defined in your `/tct/hosts` and in [ingress.yaml](https://github.com/parodos-dev/parodos/blob/main/hack/manifests/testing/ingress.yaml).
```
Comment (masayag) [inline documentation/latest/dev/guide.md:101]: pls upadte ``/tct/hosts`` to ``/etc/hosts``

File: documentation/latest/dev/guide.md (hunk 1 above)
Comment (gciavarrini) [reply documentation/latest/dev/guide.md:80]: I didn't have, prior to https://github.com/parodos-dev/parodos/pull/376 , any entry in my `/etc/hosts` 🤷

**Review by masayag — APPROVED** [review]

**Pull request merged**